seqr/models.py

```diff
@@ -1040,6 +1040,23 @@ class Meta:
         json_fields = ['guid', 'name', 'description']
 
 
+class DynamicAnalysisGroup(ModelWithGUID):
+    project = models.ForeignKey('Project', on_delete=models.CASCADE, null=True, blank=True)
+    name = models.TextField()
+    criteria = JSONField()
+
+    def __unicode__(self):
```

Comment:
Oooh, is this still needed in whatever Django version we're on?  I think this can be `__str__` in python3 if it's not a Django thing.

Reply:
its a Django thing 😿

Reply:
At some point we should probably do a django upgrade and clean up this sort of thing all over the repo